packages/relay/tests/lib/sdkClient.spec.ts

```diff
@@ -69,7 +74,7 @@ describe('SdkClient', async function () {
 
     client = client.setOperator(
       AccountId.fromString(process.env.OPERATOR_ID_MAIN!),
-      PrivateKey.fromString(process.env.OPERATOR_KEY_MAIN!),
+      PrivateKey.fromStringDer(process.env.OPERATOR_KEY_MAIN!),
```

Comment:
Q: why the change as part of this PR?
Does this force DER only?
I believe we wanted to move to a place that supported DER or hex for EVM tools.
If not related this should be reverted and tackled in another PR

Reply:
The `fromString` method was getting flagged for being deprecated by eslint, and since we use DER keys everywhere in our `.env` files I changed this to `fromStringDer`. Please let me know if I'm missing anything and I should revert this.

Reply:
Gotcha. 
It's only affecting the tests path in this case so it should be fine.
Carry on.

Reply:
Eric wrote a method to create private key based on format here https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/packages/relay/src/utils.ts#L44-L56. Can this be employed here to make it more dynamic?

Reply:
Replaced it with `createPrivateKeyBasedOnFormat `, thanks!